themes/default/content/challenge/holiday-shopping.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Holiday Shopping"
+layout: challenge/single
+description: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas 
+meta_desc: |
+    Create a simple shopping list app using AWS Fargate and MongoDB Atlas
+meta_image: /images/challenge/challenge_cta.png
+---
+
+## Welcome to the Pulumi Challenge!
```

Comment:
Now that we have several challenges, should we link to other challenges from here somehow? Even if that's just to include a sort of breadcrumb that leads back to the challenge landing page?

Reply:
Do we need to do that in this markdown file or somewhere in the layouts?

Reply:
In the layouts. I've added https://github.com/pulumi/devrel-team/issues/460 as a follow-up.